packages/blaze/domrange.js

```diff
@@ -88,34 +85,31 @@ DOMRange._insertNodeWithHooks = function (n, parent, next) {
   // `|| null` because IE throws an error if 'next' is undefined
```

Comment:
@StorytellerCZ do we still want IE support?

Reply:
I am waiting for the decision before starting implementation.